packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -527,6 +527,18 @@ const runTask = new tasks.EcsRunTask(stack, 'RunFargate', {
 });
 ```
 
+You can set the `useTaskDefinitionFamily` prop to true if you want the task to be run using latest revision
+of the task definition family:
```

Comment:
there should also be 1-2 lines outlining when / why users should specify this parameter and the impact of going with the default.